### DISCUSSION TITLE
Supervised ML SOAP dataset loading

### DISCUSSION THREAD

**Discussion by rschireman** [SINGROUP/dscribe#82] [Q&A] [answered]
Hi all,

I have a quick question about both tensorflow and pytorch examples of training an ML based force field.  In both examples, loading the numpy array for `D_numpy` and `dD_dr_numpy` is accomplished in the following manner:

`D_numpy = np.load("D.npy")[:, 0, :]  # We only have one SOAP center`
`dD_dr_numpy = np.load("dD_dr.npy")[:, 0, :, :, :]  # We only have one SOAP center`

Maybe I'm not familiar enough with the SOAP kernel, but do you always have only one SOAP center? Are there any examples without using a single SOAP center?

**Comment by lauri-codes** [comment, marked as answer]
Hi @rschireman,

One should definitely go beyond one center, and the tutorial uses one just to keep things simple. This is also mentioned in the tutorial preamble:
> SOAP descriptor calculated directly between the two atoms. Once again this is for simplicity and in real systems you would have many more centers, possibly on top of each atom.

Unfortunately, I don't have any good examples ready for multi-center learning. I would suggest looking at the literature, possibly starting with the [GAP-model](https://arxiv.org/abs/1502.01366).

Hope this helps.
